web/integration_test.go

```diff
@@ -36,21 +35,16 @@ func TestIntegration_Scheduler(t *testing.T) {
 }
 
 func TestIntegration_HelloWorld(t *testing.T) {
-	gock.EnableNetworking()
-	defer cltest.CloseGock(t)
+	tickerResponse := `{"high": "10744.00", "last": "10583.75", "timestamp": "1512156162", "bid": "10555.13", "vwap": "10097.98", "volume": "17861.33960013", "low": "9370.11", "ask": "10583.00", "open": "9927.29"}`
+	mockServer, cleanup := cltest.NewHTTPMockServer(t, 200, "GET", tickerResponse, func(body string) {})
+	defer cleanup()
```

Comment:
The second value in the tuple isn't `cleanup`, it's `assertCalled`. Let's change that name and then we're good to go 👍 .